does not compile with fmt 9.0.0

https://downloads.openwrt.org/snapshots/faillogs/mips64_octeonplus/packages/gerbera/compile.txt

Today fmt 9 is in debian unstable and I see

```
/usr/include/fmt/format.h:3950:36: error: passing ‘const fmt::v9::formatter<fmt::v9::join_view<__gnu_cxx::__normal_iterator<const ColumnUpdate*, std::vector<ColumnUpdate> >, __gnu_cxx::__normal_iterator<const ColumnUpdate*, std::vector<ColumnUpdate> >, char>, char, void>::formatter_type’ {aka ‘const fmt::v9::formatter<ColumnUpdate>’} as ‘this’ argument discards qualifiers [-fpermissive]
 3950 |       out = value_formatter_.format(map(*it), ctx);
      |             ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~
/home/marillat/src/gerbera-1.11.0/src/database/sql_format.h:64:10: note:   in call to ‘decltype (ctx.out()) fmt::v9::formatter<ColumnUpdate>::format(const ColumnUpdate&, FormatContext&) [with FormatContext = fmt::v9::basic_format_context<fmt::v9::appender, char>; decltype (ctx.out()) = fmt::v9::appender]’
   64 |     auto format(const ColumnUpdate& a, FormatContext& ctx) -> decltype(ctx.out())
      |          ^~~~~~
/usr/include/fmt/format.h:3955:38: error: passing ‘const fmt::v9::formatter<fmt::v9::join_view<__gnu_cxx::__normal_iterator<const ColumnUpdate*, std::vector<ColumnUpdate> >, __gnu_cxx::__normal_iterator<const ColumnUpdate*, std::vector<ColumnUpdate> >, char>, char, void>::formatter_type’ {aka ‘const fmt::v9::formatter<ColumnUpdate>’} as ‘this’ argument discards qualifiers [-fpermissive]
 3955 |         out = value_formatter_.format(map(*it), ctx);
      |               ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~
/home/marillat/src/gerbera-1.11.0/src/database/sql_format.h:64:10: note:   in call to ‘decltype (ctx.out()) fmt::v9::formatter<ColumnUpdate>::format(const ColumnUpdate&, FormatContext&) [with FormatContext = fmt::v9::basic_format_context<fmt::v9::appender, char>; decltype (ctx.out()) = fmt::v9::appender]’
   64 |     auto format(const ColumnUpdate& a, FormatContext& ctx) -> decltype(ctx.out())
      |          ^~~~~~
ninja: build stopped: subcommand failed.
```

--- a/src/database/sql_format.h
+++ b/src/database/sql_format.h
@@ -42,7 +42,7 @@ struct SQLIdentifier {
 template <>
 struct fmt::formatter<SQLIdentifier> : formatter<std::string_view> {
     template <typename FormatContext>
-    auto format(const SQLIdentifier& tn, FormatContext& ctx) -> decltype(ctx.out())
+    auto format(const SQLIdentifier& tn, FormatContext& ctx) const -> decltype(ctx.out())
     {
         return format_to(ctx.out(), "{}{}{}", tn.quote_begin, tn.name, tn.quote_end);
     }
@@ -61,7 +61,7 @@ struct ColumnUpdate {
 template <>
 struct fmt::formatter<ColumnUpdate> : formatter<std::string_view> {
     template <typename FormatContext>
-    auto format(const ColumnUpdate& a, FormatContext& ctx) -> decltype(ctx.out())
+    auto format(const ColumnUpdate& a, FormatContext& ctx) const -> decltype(ctx.out())
     {
         return format_to(ctx.out(), "{} = {}", a.column, a.value);
     }

